civiform/civiform
# add universal question text in program list view

### Description

This PR add text to the admin program index view that shows how many universal questions a program contains. If there are no questions set to universal, the text is not shown. This feature is hidden behind the `UNIVERSAL_QUESTIONS` feature flag.

### Checklist

#### General


#### New Features


### Instructions for manual testing

1) Log in as an admin
2) Create a program
3) Create a question (do not mark it as universal
5) Navigate to the admin program list view, you should not see any changes
6) Edit the question to mark it as universal
7) Navigate back to the program list view
8) You should see the text "Contains 0 of 1 universal questions" on the program card
9) Add the question to the program
10) Navigate back to the program list view
11) You should see the text "Contains all universal questions" on the program card


### Issue(s) this completes

Fixes #6097;
